spec/repository_spec.rb

```diff
@@ -26,3 +26,17 @@
     expect(EXTRA_FILES).to eq([])
   end
 end
+
+describe "The file \"renamed-styles.json\"" do
+  it "must be valid JSON" do
+    expect(JSON_IS_VALID).to be true
+  end
```

Comment:
> For the journals repo you could also try to do something with the jsonlint npm package

Thanks, but I think I'd like to standardize on doing all checks via RSpec across the various repos. And I want to do more checks than just validating the JSON.